app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
- [ ]  Make sure you bring all the methods related to comment model class in here, and seemed like you've added them in your post class model.